MIGRATION.md

```diff
@@ -4,20 +4,20 @@
 
 - Ground up refactor of the way the `MaterialDrawer` is used and integrated into the app.
 - Please consider carefully before upgrading to this version.
-- Prior to v8 the `Drawer` would automatically inject the `DrawerLayout` into the layout hierarchy, apply window flags, and take over control of the `ActionbarDrawerToggle`.
-  - This may seemed convenient for easy usecases, but created big problems for more advanced implementations where taking over window insets is expected
-  - v8 will no longer do any of this, and gives back all the control to the developer, no more unexpected layout flags, chagnes to the layout hierarchy or anything similar.
+- Prior to v8 the `Drawer` would automatically inject the `DrawerLayout` into the layout hierarchy, apply window flags, and take over control of the `ActionBarDrawerToggle`.
+  - This may seem convenient for easy usecases, but created big problems for more advanced implementations where taking over window insets is expected
```

Comment:
I think this should be changed (expected -> unexpected), but I was not sure so I did not include it in the PR. Let me know if I can apply this change as well.
```suggestion
  - This may seem convenient for easy usecases, but created big problems for more advanced implementations where taking over window insets is unexpected
```

Reply:
It's actually right. `it's anticipated by the user to take over window insets`. Which was not possible prior v8